Dockerfile

```diff
@@ -24,7 +23,8 @@ RUN \
     echo '' > requirements.txt && \
     pip3 install . --no-deps
 
-RUN pip install jsonformatter==0.3.2 gql[requests]==3.3.0
+RUN pip install jsonformatter==0.3.2 gql[requests]==3.3.0 \
+    Flask==2.2.* Flask-Login==0.6.* Flask-SQLAlchemy==2.5.* Flask-Migrate==4.0.* SQLALchemy==1.4.*
```

Comment:
These packages are installed by the dnf command above (though some have different versions), any reason of installing them with pip?

Reply:
I was installing them with pip in order to pin their version. The version that was available via dnf was higher, and I had to reinstall (and downgrade) them manually.
But that was before I updated the flask packages version! I removed those package installs afterwards.